enterprise/ha/src/test/java/org/neo4j/test/ha/ClusterTest.java

```diff
@@ -59,8 +60,8 @@ public void testCluster() throws Throwable
     {
         ClusterManager clusterManager = new ClusterManager( fromXml( getClass().getResource( "/threeinstances.xml" ).toURI() ),
                 testDirectory.directory(  "testCluster" ),
-                MapUtil.stringMap( HaSettings.ha_server.name(), "localhost:6001-6005",
-                                  HaSettings.tx_push_factor.name(), "2"));
+                entry( HaSettings.ha_server.name(), "localhost:6001-6005" ).
+                        entry( HaSettings.tx_push_factor.name(), "2" ).create() );
```

Comment:
Sure. Does it matter? I was using ClusterTest to try out the new setting, and did this change at the same time, but then obviously reverted the level setting so you don't see it here. The new idiom was still nice to have, but we can apply it incrementally.